api.raml

```diff
@@ -273,6 +273,22 @@ schemas:
     responses:
       200:
         description: The new user record was created.
+        body:
+          application/json:
+            description: The newly-created user record.
+            schema: User
+            example: |
+              {
+                "employee_id": "12345",
+                "first_name": "John",
+                "last_name": "Smith",
+                "display_name": "John Smith",
```

Comment:
the code to accomplish that is easy enough, @forevermatt let me know what you end up deciding here.